docs-source/content/samples/simple/azure-kubernetes-service/model-in-image.md

```diff
@@ -0,0 +1,836 @@
+---
+title: "Model in Image"
+date: 2020-11-24T18:22:31-05:00
+weight: 3
+description: "Sample for creating a WebLogic cluster on the Azure Kubernetes Service with model in image approach."
+---
+
+This sample demonstrates how to use the [Oracle WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS) using the model in image approach. After going through the steps, your WLS domain runs on an AKS cluster instance and you can manage your WLS domain by interacting with the operator.
+
+#### Contents
+
+ - [Prerequisites](#prerequisites)
+ - [Create an AKS cluster](#create-the-aks-cluster)
+ - [Install WebLogic Server Kubernetes Operator](#install-weblogic-server-kubernetes-operator)
+ - [Create Docker image](#create-docker-image)
+ - [Create WebLogic domain](#create-weblogic-domain)
+ - [Invoke the web application](#invoke-the-web-application)
+ - [Rolling updates](#rolling-updates)
+ - [Clean up resource](#clean-up-resources)
+ - [Troubleshooting](#troubleshooting)
+ - [Useful links](#useful-links)
+ 
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/prerequisites-02.txt" >}}
+
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/create-aks-cluster-body-01.txt" >}}
+
+##### Clone WebLogic Server Kubernetes Operator repository
+
+Clone the [Oracle WebLogic Server Kubernetes Operator repository](https://github.com/oracle/weblogic-kubernetes-operator) to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.1.1.
+
+```bash
+$ git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+cd weblogic-kubernetes-operator
+#TODO: we have to fix the branch after the source code are merged
```

Comment:
@edburns, we need to come up with some agreement here. The `master` branch is the latest production release plus any documentation updates or fixes, as we've discussed. I've not been pushing those same documentation fixes to every release branch (such as `release/3.1.1`) or moving the git tags.

You could instead direct customers to `git clone` and then stay on master to have the latest documentation, but then use the chart repository option: https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/installation/#alternatively-install-the-operator-helm-chart-from-the-github-chart-repository, and using the `helm install --version 3.1.1` option to use the 3.1.1 version of the chart, which will have the 3.1.1 image of the operator as the default.

My preference, though, would be to go further and find a way to do periodic integration testing so that we can just tell customers to use the latest version of the operator. We've had very few upgrade bugs (although, we did just fix one) and we are making it a priority to keep upgrading simple. Therefore, I believe that it will be safe to tell customers to use the latest as long as we do periodic tests (e.g. once per quarter).

Looping in @mriccell.